test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -4,245 +4,246 @@
-	    $expected  = "testfile1.txt:5:No matches"
+            Select-String matches (Join-Path -Path $testDirectory -ChildPath testfile1.txt)  | Should -Match $expected
+            Pop-Location
+        }
 
-	    Select-String 'matc*' $testInputFile -CaseSensitive | Should -Match $expected
-	}
+        It "Should return the fourth line in testfile1 when a regular expression is used" {
+            $expected  = "testfile1.txt:5:No matches"
 
-	It "Should return the fourth line in testfile1 when a regular expression is used, using the alias for casesensitive" {
-	    $expected  = "testfile1.txt:5:No matches"
+            Select-String 'matc*' $testInputFile -CaseSensitive | Should -Match $expected
+        }
 
-	    Select-String 'matc*' $testInputFile -ca | Should -Match $expected
-	}
+        It "Should return the fourth line in testfile1 when a regular expression is used, using the alias for casesensitive" {
+            $expected  = "testfile1.txt:5:No matches"
 
-	It "Should return all strings where 'in' is found in testfile1, when -Raw is used." {
-	    $expected1 = "This is a text string, and another string"
-	    $expected2 = "This is the second line"
-	    $expected3 = "This is the third line"
-	    $expected4 = "This is the fourth line"
+            Select-String 'matc*' $testInputFile -ca | Should -Match $expected
+        }
 
-	    (Select-String in $testInputFile -Raw)[0] | Should -BeExactly $expected1
-	    (Select-String in $testInputFile -Raw)[1] | Should -BeExactly $expected2
-	    (Select-String in $testInputFile -Raw)[2] | Should -BeExactly $expected3
-	    (Select-String in $testInputFile -Raw)[3] | Should -BeExactly $expected4
-	    (Select-String in $testInputFile -Raw)[4] | Should -BeNullOrEmpty
-	}
+        It "Should return all strings where 'in' is found in testfile1, when -Raw is used." {
+            $expected1 = "This is a text string, and another string"
+            $expected2 = "This is the second line"
+            $expected3 = "This is the third line"
+            $expected4 = "This is the fourth line"
+
+            (Select-String in $testInputFile -Raw)[0] | Should -BeExactly $expected1
+            (Select-String in $testInputFile -Raw)[1] | Should -BeExactly $expected2
+            (Select-String in $testInputFile -Raw)[2] | Should -BeExactly $expected3
+            (Select-String in $testInputFile -Raw)[3] | Should -BeExactly $expected4
+            (Select-String in $testInputFile -Raw)[4] | Should -BeNullOrEmpty
+        }
 
-	It "Should ignore -Context parameter when -Raw is used." {
-		$expected = "This is the second line"
-		Select-String second $testInputFile -Raw -Context 2,2 | Should -BeExactly $expected
-	}
+        It "Should ignore -Context parameter when -Raw is used." {
+            $expected = "This is the second line"
+            Select-String second $testInputFile -Raw -Context 2,2 | Should -BeExactly $expected
+        }
     }
     Push-Location $currentDirectory
```

Comment:
This should be in a `AfterAll{}` and at the top, not at the bottom

Reply:
added